packages/kbn-config/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationDir": "./target/types",
+    "outDir": "./target/out",
+    "stripInternal": false,
```

Comment:
Spent way to much time trying to debug the generated code, before finding that I copied a config with `"stripInternal": true,`.